auth/config.go

```diff
@@ -42,7 +42,7 @@ func DefaultConfig() Config {
 		},
 		HTTPTimeout:         30,
 		ClockSkew:           5000,
-		ContractValidators:  []string{"irma", "uzi", "dummy"},
+		ContractValidators:  []string{"irma", "uzi", "selfsigned", "dummy"},
```

Comment:
@stevenvegt does order matter here, should we add it to the end? (I think you fixed weirdness with Koanf slices a long time ago, right?)